lib/grammers-mtsender/src/utils.rs

```diff
@@ -72,7 +72,7 @@ impl Timeout {
 pub async fn sleep(duration: Duration) {
     #[cfg(not(all(target_arch = "wasm32", target_os = "unknown")))]
     {
-        return tokio::time::sleep(duration).await;
+        tokio::time::sleep(duration).await;
```

Comment:
Can you explain this change? And, if it needs to stay, please make it a new commit.

Reply:
Ah right, the cfg confused me, but yeah the parent had no return type anyway.